subliminal/video.py

```diff
@@ -147,6 +163,18 @@ def __repr__(self):
             return '<%s [%r]>' % (self.__class__.__name__, self.title)
         return '<%s [%r, %r]>' % (self.__class__.__name__, self.title, self.year)
 
+    def __hash__(self):
+        if self.year is None:
+            return hash((
+                self.title,
+                self.year,
+            ))
+        return hash(self.title)
```

Comment:
At one point i was using it for the [NZBGet plugin](nzbget.net/forum/viewtopic.php?f=8&t=1422) I wrote. It allows you to **filter()** content and eliminate duplicate videos found.  the **==** operator or **in** operator could be used when comparing a big list of videos.

I can remove it if you want; i realize your not using it internally for anything It just makes it easier for people (like me) who use your tool's lower level functions to handle batches etc.